ssn/index.html

```diff
@@ -1571,7 +1567,7 @@ <h3 id="horizontal-segmentation">Horizontal Segmentation</h3>
                       <span property="skos:example">Dividing a field site into quadrants.</span><br>
                       <span property="skos:example">Drawing blood from a patient.</span><br>
                       <span property="skos:example">Drilling an observation well.</span><br>
-                      <span property="skos:example">Establishing a station for environmental monitoring.</span><br>
+                      <span property="skos:example">Environmental monitoring.</span><br>
```

Comment:
No - don't accept this change. 'Sampling' in this case corresponds with the process of siting and establishing a monitoring station - the result is a new monitoring station, not a time-series (that comes later).